simulations/parallel/bsl_vp_3d3v_cart_dd/CMakeLists.txt

```diff
@@ -15,6 +15,91 @@ IF(USE_FMEMPOOL)
   add_definitions( -DUSE_FMEMPOOL )
 ENDIF(USE_FMEMPOOL)
 
+# build a C/C++ callable interface to simulation
+ENABLE_LANGUAGE(C)
+ENABLE_LANGUAGE(CXX)
+
+include(FortranCInterface)
+
+#TODO have this detected
+SET(CMAKE_CXX_COMPILER mpicxx)
```

Comment:
Hi @kkormann . I see you worked a lot. This cmake configuration breaks compilation on mac and probably builds with Intel compilers. I have no longer access to IPP servers so I can't fix this for now.